libs/algorithms/tests/unit/algorithms/CMakeLists.txt

```diff
@@ -140,7 +140,7 @@ foreach(test ${tests})
 
   source_group("Source Files" FILES ${sources})
 
-  set(folder_name "Tests/Unit/Parallel/Algorithms")
+  set(folder_name "Tests/Unit/Modules/Algorithms/Algorithms")
```

Comment:
Probably a typo...

Reply:
Intentional, but maybe a bad choice. The module is now called algorithms, and it has two sub-somethings called algorithms and container algorithms, so the last `Algorithms` just distinguishes between the two. I should probably just collapse them into just one namespace.

Reply:
@msimberg this should either be changed to "Tests/Modules/Unit/Algorithms/Algorithms" or we should change the folder names for the modules hpx_assertion, hpx_cache, and hpx_collectives. The same is probably true for the regression and perf tests.

Reply:
Good catch. The test names are `tests.unit.modules....` so I'd rather change the folder names in the other modules because I find it slightly more useful to do `make tests.unit` than `tests.modules`. We could of course have both but that could get messy... Any preference? If not I'll stick with `Tests/Unit/Modules` and `tests.unit.modules`.

Reply:
I have no preference, just would like to have it consistent.